src/sentry/integrations/utils/code_mapping.py

```diff
@@ -273,14 +308,15 @@ def _normalized_stack_and_source_roots(
         if source_path == stacktrace_root:
             stacktrace_root = ""
             source_path = ""
-        # stacktrace_root starts with "./"
-        elif stacktrace_root.startswith("./"):
-            without = stacktrace_root.replace("./", "")
+        # stacktrace_root starts with a FILE_PATH_PREFIX_REGEX
+        elif (without := remove_straight_path_prefix(stacktrace_root)) != stacktrace_root:
+            start_index = get_straight_path_prefix_end_index(stacktrace_root)
+            starts_with = stacktrace_root[:start_index]
```

Comment:
Modified to allow for `_normalized_stack_and_source_roots` function to work for all of the new prefixes